cmd/export.go

```diff
@@ -38,111 +27,26 @@ func setupExportCommand() *cobraext.Command {
 	exportDashboardCmd.Flags().Bool(cobraext.TLSSkipVerifyFlagName, false, cobraext.TLSSkipVerifyFlagDescription)
 	exportDashboardCmd.Flags().Bool(cobraext.AllowSnapshotFlagName, false, cobraext.AllowSnapshotDescription)
 
+	exportIngestPipelinesCmd := &cobra.Command{
+		Use: "ingest-pipelines",
+		Short: "Export ingest pipelines from Elasticsearch",
+		Long: exportIngestPipelinesLongDescription,
+		Args: cobra.NoArgs,
+		RunE: exportIngestPipelinesCmd,
+	}
+
+	exportIngestPipelinesCmd.Flags().StringSliceP(cobraext.IngestPipelineIDsFlagName, "d", nil, cobraext.IngestPipelineIDsFlagDescription)
+	exportIngestPipelinesCmd.Flags().Bool(cobraext.TLSSkipVerifyFlagName, false, cobraext.TLSSkipVerifyFlagDescription)
+	exportIngestPipelinesCmd.Flags().Bool(cobraext.AllowSnapshotFlagName, false, cobraext.AllowSnapshotDescription)
+
 	cmd := &cobra.Command{
 		Use:   "export",
 		Short: "Export package assets",
 		Long:  exportLongDescription,
 	}
 	cmd.AddCommand(exportDashboardCmd)
+	cmd.AddCommand(exportIngestPipelinesCmd)
 	cmd.PersistentFlags().StringP(cobraext.ProfileFlagName, "p", "", fmt.Sprintf(cobraext.ProfileFlagDescription, install.ProfileNameEnvVar))
 
 	return cobraext.NewCommand(cmd, cobraext.ContextPackage)
-}
-
-func exportDashboardsCmd(cmd *cobra.Command, args []string) error {
```

Comment:
Moves export dashboards functionality into own file now that we are adding support for ingest-pipelines